tox.ini

```diff
@@ -19,4 +19,9 @@ basepython = python3.7
 deps = flake8
        flake8-docstrings
        pydocstyle==3.0.0
-commands = flake8 aea examples scripts tests --exclude=.md,aea/*_pb2.py,aea/__init__.py,aea/cli/__init__.py,tests/common//oef_search_pluto_scripts --ignore=E501,E701
\ No newline at end of file
+commands = flake8 aea examples scripts tests --exclude=.md,aea/*_pb2.py,aea/__init__.py,aea/cli/__init__.py,tests/common//oef_search_pluto_scripts --ignore=E501,E701
+
+[testenv:mypy]
+basepython = python3.7
+deps = mypy
+commands = mypy aea examples
```

Comment:
Should be: mypy aea examples scripts tests

Reply:
thanks, will fix.